_file/keyevents.js

```diff
@@ -6,6 +6,10 @@ $(document).keyup(function(e){
   }
 });
 
+$(function () {
+  $('[data-toggle="tooltip"]').tooltip()
+})
+
```

Comment:
This code snippet probably shouldn't be in this file since it's about bootstrap/tooltips and not about keyevents.